programs/bpf_loader/src/lib.rs

```diff
@@ -119,9 +119,7 @@ pub fn create_and_cache_executor(
         .map_err(|e| map_ebpf_error(invoke_context, e))?;
     executable.set_syscall_registry(syscall_registry);
 
-    if invoke_context.is_feature_active(&bpf_just_in_time_compilation::id())
-        && executable.jit_compile().is_err()
-    {
+    if invoke_context.use_bpf_jit() && executable.jit_compile().is_err() {
         return Err(BPFLoaderError::JustInTimeCompilationFailed.into());
```

Comment:
I think we want to fallback to to the interpreted VM here.  Otherwise all JIT-enabled validators will fork from the rest

Reply:
Ok, so don't return an error but at least a log message?
Also, currently there is no way to check if an executable was compiled, so I would have to add that in RBPF, as there could be executables which can only be interpreted.

Reply:
> so don't return an error but at least a log message?

A metrics datapoint would be perfect

I think this part can be done as a follow-up PR though